src/components/char-check-limit/_macro-options.md

```diff
@@ -1,6 +1,6 @@
 | Name                       | Type    | Required                     | Description                                                                                                                                                                                        |
 | -------------------------- | ------- | ---------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| type                       | string  | false                        | Set to “check” to add the character limit checker to the input                                                                                                                                     |
+| variant                    | string  | false                        | Set to “check” to add the character limit checker to the input                                                                                                                                     |
 | id                         | string  | true                         | The HTML `id` of the component                                                                                                                                                                     |
 | limit                      | integer | true                         | The maximum number of characters allowed in the input                                                                                                                                              |
 | charCountPlural            | string  | true (unless `type` is set)  | The string displayed when multiple characters can be entered before the limit is reached. Set `{x}` in the string to be replaced with the number, for example “You have {x} characters remaining”. |
```

Comment:
Need to update this as type no longer exists along with the rest of the references to type?
```suggestion
| charCountPlural            | string  | true (unless `variant` is set)  | The string displayed when multiple characters can be entered before the limit is reached. Set `{x}` in the string to be replaced with the number, for example “You have {x} characters remaining”. |
```

Reply:
I dont quite understand what you mean, "Type" has been updated to "variant" here?

Reply:
Good spot! Done!